### PR TITLE
Lock10 rc updates fo Angular QS

### PR DESCRIPTION
**Pull request by christiannwamba**
- Docs updates
- Minor edits and corrections

### CODE REVIEW

**Comment by auth0build** [comment]
Can one of the admins verify this patch?

File: articles/client-platforms/angularjs/00-intro.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ There are two options for following along with these steps. You can either downl
 
 <%= include('../../_includes/_new_app') %>_
 
-![App Dashboard](/media/articles/angularjs/app_dashboard.png)
+![App Dashboard](/media/articles/angularjs/app_dashboard.̨png)
```
Comment (ntotten) [inline articles/client-platforms/angularjs/00-intro.md:20]: extra character

**Comment by mpaktiti** [comment]
@chenkie Shouldn't we close this one?

**Comment by ntotten** [comment]
Yes

**Pull request closed without merging**